src/histogram-date-range.ts

```diff
@@ -399,23 +617,54 @@ export class HistogramDateRange extends LitElement {
           width: ${this.tooltipWidth}px;
           height: ${this.tooltipHeight}px;
           top: ${-9 - this.tooltipHeight}px;
-          left: ${this.tooltipOffset}px;
-          display: ${this.tooltipVisible ? 'block' : 'none'};
+          left: ${this._tooltipOffset}px;
+          display: ${this._tooltipVisible ? 'block' : 'none'};
         }
         #tooltip:after {
           left: ${this.tooltipWidth / 2}px;
         }
       </style>
-      <div id="tooltip">${this.tooltipContent}</div>
+      <div id="tooltip">${this._tooltipContent}</div>
+    `;
+  }
+
+  private get noDataTemplate(): TemplateResult {
+    return html`
+      <div class="missing-data-message">${this.missingDataMessage}</div>
+    `;
+  }
+
+  private get activityIndicatorTemplate(): TemplateResult {
+    if (!this.loading) {
+      return html``;
```

Comment:
There's a [`nothing`](https://lit-html.polymer-project.org/guide/writing-templates#rendering-nothing) helper for when you want to output nothing.

Reply:
Cool, thanks, that's more semantic. I had tried that briefly before but it wasn't working. Realized now I just needed to add an additional return type:
```ts
  private get activityIndicatorTemplate(): TemplateResult | typeof nothing {
    if (!this.loading) {
      return nothing;
    }